### PR TITLE
chore: Setup analytics export paths

### PR DESCRIPTION
**Pull request by jimblanc**
#### Description of changes

This change sets up the initial Analytics export paths to prepare for functional APIs.

```
aws-amplify/analytics
aws-amplify/analytics/pinpoint
```

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-js/blob/main/CONTRIBUTING.md#steps-towards-contributions)
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/analytics/src/Analytics.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ import {
 	Hub,
 	parseAWSExports,
 } from '@aws-amplify/core';
-import { AWSPinpointProvider } from './Providers/AWSPinpointProvider';
+import { AWSPinpointProvider } from './providers/AWSPinpointProvider';
```
Comment (elorzafe) [inline packages/analytics/src/Analytics.ts:10]: What happen with the `Providers` directory?

**Review by elorzafe — APPROVED** [review]
Looks good, only thing is the `Providers` vs `providers` directory. It seems that could be problematic depending on the platform you are compiling?

File: packages/analytics/src/Analytics.ts (hunk 1 above)
Comment (jimblanc) [reply packages/analytics/src/Analytics.ts:10]: I just renamed it to be lower case to align with the other categories & folders in this package. No impact on the public surface.

**Comment by jimblanc** [comment]
> Looks good, only thing is the `Providers` vs `providers` directory. It seems that could be problematic depending on the platform you are compiling?

Ended up reverting this, kept running into build issues even when all imports were updated to use the lower-case directory.

**Review by elorzafe — APPROVED** [review]
Looks good!

**Review by elorzafe — APPROVED** [review]

**Review by Samaritan1011001 — APPROVED** [review]
lgtm

**Pull request merged**